src/WinGet.RestSource/Models/Schemas/Installer.cs

```diff
@@ -165,6 +166,48 @@ public Installer(Installer installer)
         /// </summary>
         public RestrictedCapabilities RestrictedCapabilities { get; set; }
 
+        /// <summary>
+        /// Gets or sets MSStoreProductIdentifier.
+        /// </summary>
+        [MSStoreProductIdentifierValidator]
+        public string MSStoreProductIdentifier { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether InstallerAbortsTerminal is set.
+        /// </summary>
+        public bool InstallerAbortsTerminal { get; set; }
+
+        /// <summary>
+        /// Gets or sets ReleaseDate.
+        /// </summary>
+        [ReleaseDateValidator]
+        public DateTime ReleaseDate { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the installer requires an install location provided.
+        /// </summary>
+        public bool InstallLocationRequired { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the installer should be pinned by default from upgrade.

```

Comment:
i believe y'all are using pinning as a technical term, but this language doesnt make much sense.

I believe if something is pinned, it will not show as an upgrade?

Reply:
Updated the description..